packages/kbn-ui-shared-deps-npm/webpack.config.js

```diff
@@ -38,6 +38,23 @@ module.exports = (_, argv) => {
         'whatwg-fetch',
         'symbol-observable',
 
+        // babel runtime helpers referenced from entry chunks
```

Comment:
How about adding a note here about the script to generate the list.

Reply:
💯